classifier/README.md

```diff
@@ -68,33 +92,50 @@ python run_classifier.py \
   --save_checkpoints_steps=500 \
   --num_train_examples=25120 \
   --num_eval_examples=500 \
-  --num_categories=3 \
+  --num_categories=2 \
   --classifier_type=Grammar
 ```
-Similarly, to train the classifier for Meaning scoring, change the training 
+Similarly, to train the meaning classifier, change the training 
 and eval files, output directory, and the classifier type correspondingly.
 
-To export the trained Grammar model, run the following. 
+To export the trained grammar classifier, run the following. 
 ```
 python run_classifier.py \
   --model_config_file=./configs/bert_config.json \
   --output_dir=${OUTPUT_DIR}/grammar_classifier \
   --do_export=true \
   --export_path=${OUTPUT_DIR}/grammar_classifier/export \
-  --num_categories=3 \
-  --classifier_type=Meaning
+  --num_categories=2 \
+  --classifier_type=Grammar
 ```
 
 ### 3. Prediction
 
-Run the following to make Grammar scoring predictions on the input file. 
+Run the following to make grammar scoring predictions on the input file. 
 ```
-! python predict_main.py \
+python predict_main.py \
  --input_file=path/to/input/file \
- --output_file=path/to/output/file \
+ --output_file=path/to/output/prediction/file \
  --vocab_file=${BERT_BASE_DIR}/vocab.txt
  --saved_model=path/to/exported/model \
  --batch_size=500 \
  --classifier_type=Grammar
 ```
-To make Meaning scoring inferences, change the classifier_type to Meaning.
+To make meaning scoring inferences, change the classifier_type to Meaning.
+
+Compute the accuracy, precision, and recall score of the predictions:
+```
+python score_main.py path/to/output/prediction/file
+```
+Example output:
+```
```

Comment:
you need to add an empty line for the block to work

Reply:
Done